BUILD_DEPS.md

```diff
@@ -0,0 +1,25 @@
+# Platform-Specific Build Dependencies
```

Comment:
Should this be moved to docs/building.md ?

Reply:
Sounds good